snuba/datasets/__init__.py

```diff
@@ -0,0 +1,18 @@
+class DataSet(object):
+    """
+    A DataSet defines the complete set of data sources, schemas, and
+    transformations that are required to:
+        - Consume, transform, and insert data payloads from Kafka into Clickhouse.
+        - Define how Snuba API queries are transformed into SQL.
+
+    This is the the initial boilerplate. schema and processor will come.
+    """
+
+    SCHEMA = None
```

Comment:
I think we should re-discuss this in the next PR when I will move the DDL stuff in here.  I am not sure it will be practical to avoid having a subclass per dataset, while we can do it for table schema